interview-questions.md

```diff
@@ -48,3 +53,14 @@ method="DELETE" /users/1          users#destroy - delete a specific user
 
 
 5. As a developer, you want to make an application that will help you manage your to do list. For the MVP, you want to create just the user's view. Create 10 user stories that will help you get your application started. 
+
```

Comment:
Good job , my only suggestion is that I would add more details for each story to make sure the person coding is following what you want